### PR TITLE
BUGFIX: TMA-1230 Log brick execution errors

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

File: lib/gooddata/lcm/lcm2.rb [hunk 1]
```diff
@@ -353,9 +353,15 @@ def perform(mode, params = {})
           success: errors.empty?
         }
 
-        has_errors = errors.any?
-        # Fail whole execution if there is any failed action
-        fail(JSON.pretty_generate(errors)) if strict_mode && has_errors
+        if errors.any?
+          error_message = JSON.pretty_generate(errors)
+          GoodData.logger.info(error_message)
```
Comment (Hahihula) [inline lib/gooddata/lcm/lcm2.rb]: Maybe the log level should be error and not info....

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1090/ : SUCCESS in 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/336/ : SUCCESS in 8m 10s

File: lib/gooddata/lcm/lcm2.rb (hunk 1 above)
Comment (sadam21) [reply lib/gooddata/lcm/lcm2.rb]: Good point, fixed.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1091/ : SUCCESS in 11s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/337/ : SUCCESS in 8m 09s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1092/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/338/ : SUCCESS in 8m 02s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/184/ : SUCCESS in 28s

**Pull request merged**